qiskit/wrapper/defaultqiskitprovider.py

```diff
@@ -88,38 +88,51 @@ def available_backends(self, filters=None):
 
         return backends
 
-    def aliased_backend_names(self):
+    def grouped_backend_names(self):
         """
-        Aggregate alias information from all providers.
+        Aggregate group names from all providers.
 
         Returns:
-            dict[str: list[str]]: aggregated alias dictionary
+            dict[str: list[str]]: aggregated group dictionary
 
         Raises:
-            ValueError: if a backend is mapped to multiple aliases
+            ValueError: if a backend is mapped to multiple groups
         """
-        aliases = {}
+        groups = {}
         for provider in self.providers:
-            aliases = {**aliases, **provider.aliased_backend_names()}
-        for pair in combinations(aliases.values(), r=2):
+            groups = {**groups, **provider.grouped_backend_names()}
+        for pair in combinations(groups.values(), r=2):
             if not set.isdisjoint(set(pair[0]), set(pair[1])):
-                raise ValueError('duplicate backend alias definition')
+                raise ValueError('duplicate backend group definition')
 
-        return aliases
+        return groups
 
     def deprecated_backend_names(self):
         """
         Aggregate deprecated names from all providers.
 
         Returns:
-            dict[str: list[str]]: aggregated alias dictionary
+            dict[str: list[str]]: aggregated dictionary of deprecated names
         """
         deprecates = {}
         for provider in self.providers:
             deprecates = {**deprecates, **provider.deprecated_backend_names()}
 
         return deprecates
 
+    def aliased_backend_names(self):
+        """
+        Aggregate aliase names from all providers.
+
+        Returns:
+            dict[str: str]: aggregated alias dictionary
+        """
+        aliases = {}
+        for provider in self.providers:
+            aliases = {**aliases, **provider.aliased_backend_names()}
```

Comment:
```python
aliases.update(provider.aliased_backend_names())
```
is faster and avoid creating a new dict in each iteration.

Reply:
sounds good.